src/app/components/mainpage/mainpage.component.html

```diff
@@ -27,14 +27,14 @@
   </div>
```

Comment:
I accidentally included this in this PR. But the work still stands and is reflected in the screenshots for https://github.com/schnarbiemeows/interview-ui/pull/1